src/main/resources/db.changeLog/changeLog-master.xml

```diff
@@ -2,7 +2,37 @@
 <databaseChangeLog
         xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:pro="http://www.liquibase.org/xml/ns/pro"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
-    <includeAll path="com/example/changelog/"/>
-</databaseChangeLog>
\ No newline at end of file
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+		http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+		http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+    <preConditions>
+        <runningAs  username="bestuser"/>
+    </preConditions>
+    <changeSet id ="1" author="bestuser">
```

Comment:
Let's move this `changeSet` with 2 `createTable` statements to separate changelog file, so `changeLog-master.xml` will have only `<include ... />`  statements.

Also let's merge all changelog files into one file, because all code is wrote in scope of one task. Changelog files can contain multiple changeSets.

You can follow the next rule: 1 task = 1 changelog file with multiple changesets.